packages/iov-jsonrpc/src/types.ts

```diff
@@ -11,7 +11,7 @@ export interface JsonRpcRequest {
   readonly params: JsonCompatibleArray | JsonCompatibleDictionary;
 }
 
-export interface JsonRpcResponse {
+export interface JsonRpcSuccessResponse {
   readonly jsonrpc: "2.0";
   readonly id: number;
```

Comment:
If you want to support tendermint, etc... id is a string
Is it defined to be a number in the jsonrpc spec?

Reply:
Oh, seems like both are valid:

> An identifier established by the Client that MUST contain a String, Number, or NULL value if included. If it is not included it is assumed to be a notification. The value SHOULD normally not be Null [1] and Numbers SHOULD NOT contain fractional parts [2]

https://www.jsonrpc.org/specification#request_object

Will generalize when using this for tendermint